.github/workflows/deploy-infra.yaml

```
@@ -604,10 +608,136 @@ jobs:
       ### END OF INSTALL AND CONFIGURE FLUX V2 ###
       ###
 
+  deploy-monitoring:
+    name: Setup logging and monitoring
+    runs-on: ubuntu-20.04
+    needs: [set-environments, check-secrets, create-eks-cluster, configure-cluster]
+    if: always() && github.event.inputs.workflow_actions != 'configure cluster' && (needs.configure-cluster.result == 'success' || needs.configure-cluster.result == 'skipped' && (needs.create-eks-cluster.result == 'success' || needs.create-eks-cluster.result == 'skipped')) 
```

A bit nitpicky, but I think it would be good to order the logical comparison in order of how they start:

```suggestion
    if: always() && github.event.inputs.workflow_actions != 'configure cluster' && (needs.create-eks-cluster.result == 'success' || needs.create-eks-cluster.result == 'skipped' && (needs.configure-cluster.result == 'success' || needs.configure-cluster.result == 'skipped')) 
```

---

sounds good. This if statement is very complicated, that might help with readability